docs/dictionary/command/mobileAdCreate.lcdoc

```diff
@@ -22,53 +22,36 @@ The name of the ad to create. This is used to reference the ad
 throughout its lifetime.
 
 type (enum):
-The type of the ad. The possible ad types are
+The type of the ad. The possible ad types are:
 
-- "text"
-- "full screen"
-
-
--   "banner": This is the default.
+- 'text'
+- 'full screen'
+- 'banner' - This is the default.
 
 
 topLeft:
-The location in pixel of the top left corner of the ad (coordinates are
-in left,top - x,y order). The default is: 0,0
+The location of the top left corner of the ad (coordinates are
+in left,top - x,y order) in pixels. The default is: 0,0
 
 metaData (enum):
```

Comment:
Since this is not actually an enum, I think the `(enum)` should be removed here. There is a way to format array variables, but for now I would just keep it as you've done it with the enum removed.